tcconfig-schema/src/main/xsd/terracotta.xsd

```diff
@@ -166,7 +178,24 @@
     <xs:attribute name="name" type="xs:string" use="required"/>
     <xs:attribute name="value" type="xs:string" use="required"/>
   </xs:complexType>
-  
+
+  <xs:complexType name="failover-priority">
+    <xs:choice>
+      <xs:element name="consistency" type="tc:consistency" minOccurs="0"/>
+      <xs:element name="availability" type="xs:string" fixed=""/>
```

Comment:
can you explain what `fixed` means?

Reply:
This is just a trick that I used define this as an empty element that does not take any attributes or sub-elements.